image/correction/main.py

```diff
@@ -0,0 +1,30 @@
+import matplotlib.pyplot as plt
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.